src/web/Select.tsx

```diff
@@ -37,12 +37,35 @@ export function Select({
     },
     []
   )
-  React.useEffect(() => void onChange?.(active), [active])
+  const [isPointerDown, setDown] = React.useState(false)
+  const [dragging, setDragging] = React.useState(false)
+  const onPointerMove = React.useCallback(() => {
+    setDragging(isPointerDown)
```

Comment:
i feel a little uneasy about setStating react 60 times a second here. that will have performance implications. is there a better way? can we use refs?

```jsx
const dragging = useRef(false)

...
dragging.current = true
```

Reply:
Hi, thanks for your review.

> can we use refs?

Yes, it will be better

> is there a reason for this?

Because when you drag transforms, the onClick will still be triggered. So I make it a flag to judge whether current event is a `click` or `drag`

Reply:
fixed